.gitignore

```diff
@@ -6,6 +6,7 @@
 /*/doc/
 /*/test/tmp/
 /.bundle
+vendor/bundle
```

Comment:
This is personal setup, not something that the project generates, so let's remove from the PR since there is no need for this line.

Reply:
Done.

Re `.gitignore`, if one has done `bundle install` in their working directory, moving to main to rebase (obviously) shows all the gem files as changes.  `vendor/bundle` is a pretty standard folder for that purpose.